### PR TITLE
Release 0.17.0

### PR DESCRIPTION
**Pull request by ywkim312**

### CODE REVIEW

**Review by pengyin-shan — APPROVED** [review]
Looks good to me. Approving

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.17.0] - 2024-08-16
```
Comment (sandeep-ps) [inline CHANGELOG.md:7]:
```suggestion
## [0.17.0] - 2024-08-28
```

File: CHANGELOG.md
Comment (sandeep-ps) [inline CHANGELOG.md:1]: Please add link for version comparison in the CHANGELOG (line number 180.)
```
[0.17.0]: https://github.com/policy-design-lab/pdl-api/compare/0.16.0...017.0
```

File: CHANGELOG.md (hunk 1 above)
Comment (ywkim312) [reply CHANGELOG.md:7]: updated

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**